Automattic/wp-calypso
# Remove pricing page deprecated variant SPP

### Changes proposed in this Pull Request

This PR removes the code related to the SPP variant of the pricing page, which didn't lead to a conclusive experiment.

### Testing instructions

- Download the PR and run cloud or Calypso
- Visit the pricing page locally
- Open a new tab and visit the pricing page in staging or production
- Check that both pages look and behave similarly
- Verify there's no error in the console
